soc/ite/ec/Kconfig.soc

```diff
@@ -7,4 +7,8 @@ config SOC_FAMILY_ITE_EC
 config SOC_FAMILY
 	default "ite_ec" if SOC_FAMILY_ITE_EC
 
+choice PM_POLICY
+	default PM_POLICY_CUSTOM if SOC_FAMILY_ITE_EC
```

Comment:
note that this enables the option for all ITE families, not only it82xx2, ITE folks is that a reasonable thing?

Reply:
Yes, this approach makes sense.